build.gradle

```diff
@@ -44,3 +45,35 @@ application {
     // Define the main class for the application.
     mainClass = 'com.whalespottingjava.WhaleSpottingJavaApplication'
 }
+
+spotless {
+  ratchetFrom 'origin/main'
```

Comment:
We're ratcheting against changes from origin/main so that we gradually fix files touched on new branches / pull requests